tsdate/evaluation.py

```diff
@@ -274,16 +274,28 @@ def node_spans(ts):
     return child_spans
 
 
+def total_span(ts):
+    """
+    Returns the total length of all "node spans", computed from
+    `node_spans(ts)`.
+    """
+    ts_node_spans = node_spans(ts)
+    ts_total_span = np.sum(ts_node_spans)
+    return ts_total_span
+
+
 def tree_discrepancy(ts, other):
     """
     For two tree sequences `ts` and `other`,
-    this method returns two values, as a tuple:
-    1. the fraction of the total span of `ts` over which each nodes' descendant
+    this method returns three values, as a tuple:
+    1. The fraction of the total span of `ts` over which each nodes' descendant
     sample set does not match its' best match's descendant sample set.
     2. The root mean squared difference
     between the times of the nodes in `ts`
     and times of their best matching nodes in `other`,
     with the average weighted by the nodes' spans in `ts`.
+    3. The proportion of the span in `ts` that is correctly
+    represented in `other`.

```

Comment:
Hm, I think what we talked about was the other way around: 
```suggestion
    3. The proportion of the span in `other` that is correctly
    represented in `ts`.

```